app/views/idv/how_to_verify/show.html.erb

```diff
@@ -30,7 +30,8 @@
             <span class="usa-tag usa-tag--informative">
               <%= t('doc_auth.tips.most_common') %>
             </span>
-            <h2 class="h3"><%= t('doc_auth.headings.verify_online') %></h2>
+            <br />
+            <label><%= t('doc_auth.headings.verify_online') %></label>
```

Comment:
Is this a `<label>` nested inside a `<label>` ? If so, is that valid HTML, or would it pose a challenge for someone using assistive technology to navigate?

Reply:
Previously we nested h2/h3 tags. I think all three are not valid. I removed all tags. I tested it using VoiceOver and I think this is a better experience. Thanks for calling that out! 👏